src/_premake_init.lua

```diff
@@ -268,6 +286,22 @@
 	}
 
 	api.register {
+		name = "debugtoolargs",
+		scope = "config",
+		kind = "list:string",
+		tokens = true,
+		pathVars = true,
+	}
+
+	api.register {
+		name = "debugtoolcommand",
+		scope = "config",
+		kind = "path",
+		tokens = true,
+		pathVars = true,
+	}
```

Comment:
Not really. 'debugger' says high level information about which debugger is in use, so everything can be configured accordingly. Some toolset's have debugger's in awkward places; NaCl for instance, it's gdb binary is separate from the various arch toolchains, and its name doesn't follow typical GCC convention. So in that case, it's necessary to specify the debugger binary explicitly.

That said, I'm still not super jazzed about the api name.